readme.md

```diff
@@ -298,6 +298,10 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [upf-bpf](https://github.com/navarrothiago/upf-bpf) - An in-kernel solution based on XDP for 5G UPF.
 - [redbpf](https://github.com/foniod/redbpf) - Tooling and framework to write eBPF code in Rust efficiently.
 - [pixie](https://github.com/pixie-io/pixie) - Observability for Kubernetes using eBPF. Features include protocol tracing, application profiling, and support for distributed bpftrace deployments.
+- [Project Calico](https://projectcalico.docs.tigera.io/about/about-calico) - Calico is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. Calico's eBPF data plane delivers blazing fast performance with true cloud-native scalability.
+ - [About eBPF technology](https://projectcalico.docs.tigera.io/about/about-ebpf)
+ - [Getting started with Calico on Kubernetes](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart)
+ - [Enabling eBPF data plane with Calico](https://projectcalico.docs.tigera.io/maintenance/ebpf/enabling-bpf)
```

Comment:
Let's keep the link for enabling eBPF in Calico.

The other two, I think, are not necessary here. Introductions to eBPF are not lacking on this list, and this one does not bring much that the others do not have. The link to the “Getting started” guide is not directly related to eBPF; I understand that it is a necessary step to deploy Calico with eBPF on a cluster, but it should be easy enough to find from the other two links. What do you think?

Reply:
agreed, and done.

Reply:
@qmonnet thanks for the comments, hopefully addressed to your satisfaction. I'm happy to squash the git commit logs if that's preferred.

Reply:
Please do, otherwise I can also do when merging.
One thing, can you please rebase your PR on the master branch? There's a conflict with a commit that got in yesterday.

Thanks for addressing it all!

Reply:
@qmonnet would you mind squashing at merge time? I resolved the merge conflict (Falco came in after Calico).